include/itkTorchImageToImageFilter.hxx

```diff
@@ -18,48 +18,77 @@
 #ifndef itkTorchImageToImageFilter_hxx
 #define itkTorchImageToImageFilter_hxx
 
-#include "itkTorchImageToImageFilter.h"
+#include <torch/torch.h>
 
-#include "itkImageRegionIterator.h"
-#include "itkImageRegionConstIterator.h"
+#include "itkTorchImageToImageFilter.h"
 
 namespace itk
 {
 
-template <typename TInputImage, typename TOutputImage>
-TorchImageToImageFilter<TInputImage, TOutputImage>
-::TorchImageToImageFilter()
-{}
+template< typename TInputTorchImage, typename TOutputImage >
+void
+TorchImageToImageFilter< TInputTorchImage, TOutputImage >
+::PrintSelf( std::ostream &os, Indent indent ) const
+{
+  Superclass::PrintSelf( os, indent );
+}
+
+
+template< typename TInputTorchImage, typename TOutputImage >
+void
+TorchImageToImageFilter< TInputTorchImage, TOutputImage >
+::GenerateInputRequestedRegion()
+{
+  // Is this correct?!!!  Instead, is the superclass implementation sufficient?!!!
+  // Call the superclass's implementation of this method
+  Superclass::GenerateInputRequestedRegion();
+
+  // Get pointers to the input image.
+  typename Self::InputTorchImageType * const inputTorchImage = const_cast< typename Self::InputTorchImageType * >( this->GetInput( 0 ) );
+
+  if( inputTorchImage != nullptr )
```

Comment:
Should go into [VerifyPreconditions()](https://itk.org/Doxygen/html/classitk_1_1ProcessObject.html#a73000f23064b7d2898fa21135080b4cf).

Reply:
Will do.